core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -228,7 +228,7 @@ object TopicCommand extends Logging {
       if (topic.partitions.exists(partitions => partitions < 1))
         throw new IllegalArgumentException(s"The partitions must be greater than 0")
 
-      if (!adminClient.listTopics().names().get().contains(topic.name)) {
+      try {
```

Comment:
removed this check and instead let the request go to the server and error (we need to handle that anyway)